### PR TITLE
Internal: Allow themes to take control of kit install [TMZ-105][ED-16046]

### PR DESCRIPTION
**Pull request by nicoladj77**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Allow theme to specify a url to be redirected after the import flow of a kit

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by louiswol94 — DISMISSED** [review]

File: app/modules/import-export/assets/js/pages/import/import-process/import-process.js [hunk 1]
```diff
@@ -136,13 +150,17 @@ export default function ImportProcess() {
...
+					return;
+				}
 				navigate( '/import/complete' );
 			} else if ( 'apply-all' === importContext.data.actionType ) { // Forcing apply-all kit content.
 				if ( kitState.data?.manifest?.plugins || importContext.data.uploadedData?.manifest.plugins ) {
 					importContext.dispatch( { type: 'SET_PLUGINS_STATE', payload: 'have' } );
 				}
 				if ( uploadedData.conflicts && Object.keys( uploadedData.conflicts ).length && ! isResolvedData ) {
-					navigate( '/import/resolver' );
+					navigate( '/import/resolver?return_to=' + returnTo );
```
Comment (louiswol94) [inline app/modules/import-export/assets/js/pages/import/import-process/import-process.js:163]: @nuritsha @nicoladj77 Will this add the param even if `returnTo` is an empty string?

File: app/modules/import-export/assets/js/pages/import/import-resolver/import-resolver.js [hunk 2]
```diff
@@ -54,7 +56,7 @@ export default function ImportResolver() {
 					color="primary"
 					onClick={ () => {
 						eventTracking( 'kit-library/approve-selection' );
-						const url = importContext.data.plugins.length ? 'import/plugins-activation' : 'import/process';
+						const url = importContext.data.plugins.length ? 'import/plugins-activation' : 'import/process?return_to=' + returnToParam;
```
Comment (louiswol94) [inline app/modules/import-export/assets/js/pages/import/import-resolver/import-resolver.js:59]: Same question here.

**Comment by louiswol94** [comment]
Question from @MichaelLiamin:
Do we have existing Playwright tests for the Kit import flow? If we have, the fact that they are not failing is a good sign, if not, can you add a test for this?

File: app/modules/import-export/assets/js/pages/import/import-resolver/import-resolver.js (hunk 2 above)
Comment (nicoladj77) [reply app/modules/import-export/assets/js/pages/import/import-resolver/import-resolver.js:59]: @louiswol94 reafctored

File: app/modules/import-export/assets/js/pages/import/import-process/import-process.js (hunk 1 above)
Comment (nicoladj77) [reply app/modules/import-export/assets/js/pages/import/import-process/import-process.js:163]: @louiswol94 refactored

**Review by louiswol94 — APPROVED** [review]

**Review by arielk — APPROVED** [review]

**Pull request merged**